schema.sql

```diff
@@ -1,5 +1,10 @@
-/* Database schema to keep the structure of entire database. */
+CREATE TABLE animals
+(
+    id integer,
```

Comment:
- [ OPTIONAL ] I think it would be great if you add to the id column not null or assigned it as primary key since the id it should be unique and not redundant to you can add it with alter command check this [article](https://www.w3schools.com/sql/sql_alter.asp) to see more about how to do it 💯🔥✨🤩